.github/workflows/release_prepare_pr.yaml

```diff
@@ -0,0 +1,99 @@
+name: Prepare Release PR
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  # Do not cancel jobs on main by forcing a unique group name
+  group: ${{ github.workflow }}-${{ github.ref_name == 'main' && github.run_id || github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  prepare_release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: main
```

Comment:
Do we need this? We get an automatic `ref` which should be correct.

Reply:
It simplifies testing from other branch